### PR TITLE
adding level_order #401

### PR DESCRIPTION
**Pull request by Jkp2072**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#401

### CODE REVIEW

**Comment by Jkp2072** [comment]
@Kajol-Kumari  please do the review

**Comment by Jkp2072** [comment]
please review it @plazzy99  @Kajol-Kumari  @SKAUL05

File: C-Plus-Plus/graphs/level_order_traversal.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ {
+     int val;        // value of node 
+     node *left;
+     node *right; // left child and right child.
+   
+ };
+ 
+ node * root; // to create nodes.
+ node * create(int n){
+     node * x =new node();
```
Comment (SKAUL05) [inline C-Plus-Plus/graphs/level_order_traversal.cpp]: Use meaningful variable names instead of alphabets

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Comment by Jkp2072** [comment]
changes resolved.

File: C-Plus-Plus/graphs/level_order_traversal.cpp [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+ root->left->right=create(5);
+  root->right->left=create(7);
+ root->right->right=create(8); // level 3
+ 
+ root->right->left->left=create(6); //level 4;
+
+ // this is the same example as above mentioned.
+ 
+ anss=level(root);
+ for(auto i : anss) cout<<i<<" ";
```
Comment (SKAUL05) [inline C-Plus-Plus/graphs/level_order_traversal.cpp]: Replace i

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/graphs/level_order_traversal.cpp (hunk 2 above)
Comment (Jkp2072) [reply C-Plus-Plus/graphs/level_order_traversal.cpp]: okay sir

**Comment by Jkp2072** [comment]
changes done

**Comment by vatsalkesarwani12** [comment]
mention issue no.

**Comment by Jkp2072** [comment]
Do I have to mention it in code?? It is 401 which is in pr heading

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
add issue number

**Comment by Jkp2072** [comment]
issue number added

File: C-Plus-Plus/graphs/level_order_traversal.cpp [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    return ans; // it will return the node values in level order.
+}
+// driver code......
+// to create a treee as per above example.
+int main()
+{
+    vector<int>anss;
+ node * root =create(1); // level 1
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/graphs/level_order_traversal.cpp:74]: take dynamic input

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**